content/docs/custom-plugins/proxy/index.md

```diff
@@ -0,0 +1,94 @@
+---
```

Comment:
Maybe add a mention and a link to the IP allowlist somewhere on this page to possibly tell customers that this is the way they could self-host custom backend plugin in their own infra

Reply:
Will do thanks.